app/site/controllers.py

```diff
@@ -52,6 +51,7 @@ def logout():
       302:
```

Comment:
Extra comment: description says "loads home page if user already login" - is that correct? That seems weird. (Also note incorrect english grammar: if user already log**ged** in"